src/main.rs

```diff
@@ -209,12 +206,16 @@ async fn handle_ldk_events(
 			network_update: _,
 			all_paths_failed,
 			path: _,
+			short_channel_id,
 		} => {
 			print!(
 				"\nEVENT: Failed to send payment{} to payment hash {:?}: ",
 				if *all_paths_failed { "" } else { " along MPP path" },
 				hex_utils::hex_str(&payment_hash.0)
 			);
+			if let Some(scid) = short_channel_id {
+				print!(" because of failure at channel {}", scid);
```

Comment:
I think the leading space should be a trailing space

Reply:
Once I moved the colon does it make more sense now?